src/util/shuffleArray.ts

```diff
@@ -0,0 +1,9 @@
+export const shuffleArray = <T>(array: T[]): T[] => {
```

Comment:
```suggestion
export const shuffleArray = <T>(array: readonly T[]): T[] => {
```

since you're not modifying the input